docker-image-src/common/neo4jlabs-plugins.json

```diff
@@ -1,13 +1,12 @@
 {
   "apoc": {
-    "versions": "https://github.com/neo4j/apoc/blob/master/versions.json",
+    "location": "/var/lib/neo4j/labs/apoc-*-core.jar",
     "properties": {
       "dbms.security.procedures.unrestricted": "apoc.*"
     }
   },
   "apoc-core": {
     "location": "/var/lib/neo4j/labs/apoc-*-core.jar",
-    "versions": "https://github.com/neo4j/apoc/blob/master/versions.json",
     "properties": {
       "dbms.security.procedures.unrestricted": "apoc.*"
     }
```

Comment:
The "apoc-core" config option was introduced by Reneta in 2021 and doesn't seem like it was ever intended to fetch from a remote URL. It's possible that this is because this config was only ever intended to work with Neo4j images that already contain APOC (4.2.x+, 4.3.x+, 4.4.x+, 5.x+)?